templates/index.html

```diff
@@ -50,12 +50,34 @@
 <body id="home-page">
   <div id="container">
     <h1 id="app-title">Iqra</h1>
-    <a id="play-link" href="https://play.google.com/store/apps/details?id=com.mmmoussa.iqra&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"><img alt="Get it on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge-border.png" /></a>
+    <a id="app-store-link" target="_blank" href="https://itunes.apple.com/us/app/iqra/id1391009396"><img alt="Get it on the Apple Store" src="/static/img/app-store-badge.png"></a>
```

Comment:
The image you're using here is low resolution. Can you instead use [the svg provided by Apple](https://developer.apple.com/app-store/marketing/guidelines/#section-badges)?